src/message/messageActionSheet.js

```diff
@@ -349,6 +350,18 @@ export const showActionSheet = (
         : {}),
       options: optionCodes.map(code => _(allButtons[code].title)),
       cancelButtonIndex: optionCodes.length - 1,
+      containerStyle: {
+        backgroundColor:
+          params.backgroundData.theme === 'night'
+            ? themeColors.night.backgroundColor
+            : themeColors.light.backgroundColor,
```

Comment:
@ray-kraesig [said](https://github.com/zulip/zulip-mobile/pull/4055#pullrequestreview-402287441):
> I'm really not a fan of the fact that the theme-colors have to be fed to two different calls of `showActionSheetWithOptions` by two different clients of this API which obtain them in two different ways, but I'm not seeing an easy way to simplify that. @chrisbobbe, you've been looking at themes and `Context`s lately; anything obvious I'm missing?

No, I don't see anything obvious you're missing, nor an easy way to simplify that inconsistency.

I'm curious how you would weigh the problem of that inconsistency against the following:

I think that, the farther removed you are from a React component, in the number of function calls, I guess, the less sense it makes to thread through bits of props/state/context, which are React-specific. (Try, if you like, chasing down where the `_` param of `copyToClipboard`, in this same file, is supposed to come from, with no prior knowledge of the plumbing!)

With this in mind, I think it's nice to avoid a heroic effort to get `this.context.backgroundColor` passed all the way down here.

The following is outside the scope of this PR; I wouldn't ask for any of it to be included here.

[...still typing]

Reply:
(Of course I'm willing to be convinced that my preferences, implicit or explicit, are wrong. 🙂 )

Reply:
I definitely want to respond to some of this, but as you note, it's outside the scope of this PR. I'll ping you in chat!

> If there were, wouldn't it be nice to do something like (hand-wavey pseudocode)
> 
> ```
> backgroundColor: ThemeContext.get('backgroundColor')
> ```
> 
> right there, in showActionSheetWithOptions, with no boilerplate except an import of ThemeContext?

Yes; this is pretty much exactly what I was hoping you'd be able to say existed. Oh well!